chef/data_bags/crowbar/bc-template-glance.json

```diff
@@ -54,6 +54,7 @@
       },
       "rbd": {
         "store_ceph_conf": "/etc/ceph/ceph.conf",
+        "store_admin_keyring": "/etc/ceph/ceph.client.admin.keyring",
```

Comment:
I don't think we should set this by default. And if we make it optional it's not required to be set.